src/App.js

```diff
@@ -23,7 +24,7 @@ function App() {
     <Routes>
     <Route path="/reviews" element={<ReviewsPage/>}></Route>
     <Route path="/reviews/:category" element={<ReviewsPage/>}></Route>
-     
+    <Route path="/review/:review_id" element={<ReviewPage/>}></Route>
```

Comment:
I know this is picky but I think I would have gone for
categories/:category
reviews/:review_id
rather than the paths you've chosen here but its totally fine